Sources/MessageInputBarDelegate.swift

```diff
@@ -24,8 +24,14 @@
 
 import Foundation
 
+@objc
 public protocol MessageInputBarDelegate: class {
-
-    func sendButtonPressed(sender: UIButton, textView: UITextView)
-
+    
+    @objc optional func inputBar(_ inputBar: MessageInputBar, didSelectSendButtonWith text: String)
+    
+    @objc optional func inputBar(_ inputBar: MessageInputBar, didChangeIntrinsicContentTo size: CGSize)
+    
+    @objc optional func inputBar(_ inputBar: MessageInputBar, textViewTextDidChangeTo text: String)
+    
+    @objc optional func inputBar(_ inputBar: MessageInputBar, didSwipeTextViewWith gesture: UISwipeGestureRecognizer)
 }
```

Comment:
Instead of marking these with the `@objc` attribute to make them optional. Can we just provide a default implementation via a protocol extension that is a no-op?

Reply:
Didn't know that was possible! Changed it to that